File: app/src/main/java/com/vladislav/workoutassistant/data/db/dao/SetVsExerciseMatchingDao.java
package com.vladislav.workoutassistant.data.db.dao;

import com.vladislav.workoutassistant.data.db.entity.SetVsExerciseMatching;

import java.util.List;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.OnConflictStrategy;

@Dao
public interface SetVsExerciseMatchingDao {

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    void insertMatching(List<SetVsExerciseMatching> matching);
}

File: app/src/main/java/com/vladislav/workoutassistant/data/db/dao/SetVsExerciseMatchingDao.kt
package com.vladislav.workoutassistant.data.db.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import com.vladislav.workoutassistant.data.db.entity.SetVsExerciseMatching

@Dao
interface SetVsExerciseMatchingDao {

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun insertMatching(matching: List<SetVsExerciseMatching>)
}
